libs/sdk-ui-dashboard/src/presentation/dashboard/Dashboard.tsx

```diff
@@ -255,7 +257,18 @@ const DashboardHeader = (props: IDashboardProps): JSX.Element => {
 
 const DashboardInner: React.FC<IDashboardProps> = (props: IDashboardProps) => {
     const locale = useDashboardSelector(selectLocale);
-    const onFiltersChange = useDispatchDashboardCommand(changeFilterContextSelection);
+    const filtersChange = useDispatchDashboardCommand(changeFilterContextSelection);
+    const resetDateFilter = useDispatchDashboardCommand(clearDateFilterSelection);
+
+    const onFiltersChange = (filters: IDashboardFilter[], correlationId?: string | undefined) => {
+        const hasDateFilter = filters.some(isDateFilter);
```

Comment:
This should probably be better fixed directly in the changeFilterContextSelectionHandler, not here.